repository-data/local/src/main/resources/hcm-config/configuration/groups/editor.yaml

```diff
@@ -0,0 +1,6 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:groups/editor:
+      hipposys:members:
+      - unlocker
```

Comment:
I'm guessing that this does _not_ mean that every member of group editor gets 'unlocker' privileges, right?